spec/routes/runtime/spec_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require_relative "../spec_helper"
+
+RSpec.configure do |config|
+  config.before {
+    allow(Config).to receive(:clover_runtime_token_secret).and_return(Config.clover_session_secret)
+  }
+end
+
+def login_runtime
+  vm = Vm.create_with_id(display_state: "running", unix_user: "x", public_key: "x",
+    name: "x", family: "standard", cores: 2, location: "x", boot_image: "x")
+  header "Authorization", "Bearer #{vm.runtime_token}"
+  vm
```

Comment:
What about passing vm to this function and login with the passed vm's runtime_token (by also changing the function's name). Returning vm from login_runtime looks counter-intuitive to me.

Reply:
Fair point. I change it